iidm/iidm-serde/src/test/resources/V1_12/areaRoundTripRef.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you adapt this file and the V1_11 one to use the same network as for v1.13?
It would make it easier to spot the differences between each version.

Reply:
adressed in e8eaddeddefc0cbe61a1af7eb490f0d23305bc5c